src/tentaclio/clients/google_drive_client.py

```diff
@@ -187,26 +194,49 @@ def is_file(self):
         return not self.is_dir
 
 
-class _ListFilesRequest(_GoogleDriveRequest):
+class _Lister(
+    _GoogleDriveRequest, abc.ABC, Generic[T],
+):
     def __init__(self, service: Any, **kwargs):
         super().__init__(service, kwargs)
         # some standard arguments to send to the service
         self.args["pageSize"] = 100
-        self.args["fields"] = "files(id, name, mimeType, parents)"
 
-    def list(self, url_base: Optional[str] = None) -> Iterable[_GoogleFileDescriptor]:
+    def list(self, url_base: Optional[str] = None) -> Iterable[T]:
```

Comment:
this method doesn't actually use `url_base` (it's left to the constructor of `_ListDrivesRequest` only)

Reply:
awesome catch, thanks!